### PR TITLE
Plat 665 cli v2

### PR DESCRIPTION
**Pull request by michaelloosen**
This PR includes the changes required in the Platform to start making use of CLIv2

### CODE REVIEW

**Review by Mark-Labuschagne — APPROVED** [review]

File: get-cli.sh [hunk 1]
```diff
@@ -6,30 +6,30 @@ cli_version=${2:-latest}
 case ${option} in
```
Comment (nour-borgi) [inline get-cli.sh:6]: Aren't we going to update the package-base image version in Dockerfile as well?

Comment (michaelloosen) [reply get-cli.sh:6]: We haven't changed anything that relates to the Docker image, so I don't think we need to update the image version in the Dockerfile?

Thanks, yeah, I missed the other files, will make the updates.

Comment (michaelloosen) [reply get-cli.sh:6]: @nour-borgi I think I got them all, can you please just re-review this PR when you have a chance?

**Review by nour-borgi — APPROVED** [review]
LGTM

**Pull request merged**